programs/aes/aescrypt2.c

```diff
@@ -80,6 +80,18 @@ int main( void )
 }
 #else
 
+#if defined(MBEDTLS_ENTROPY_HARDWARE_ALT)
```

Comment:
Question - Why were all these additions to the samples necessary?

This should only be needed in an Mbed OS build, and in that we won't build the `programs`.

Reply:
To make CI happy when `MBEDTLS_ENTROPY_HARDWARE_ALT` is defined. (The CMake build builds also the programs.)